test/fixtures/poro.rb

```diff
@@ -1,6 +1,10 @@
 class Model
   FILE_DIGEST = Digest::MD5.hexdigest(File.open(__FILE__).read)
 
+  def self.create(params)
+    @attributes = params
+  end
```

Comment:
elsewhere in this PR we have

```ruby
 @post = Post.create(post_params)
```

which leads me to believe `create` should be `new(params)`, no?